src/main/java/org/shanerx/tradeshop/data/config/Setting.java

```diff
@@ -69,7 +69,7 @@ public enum Setting {
     SHOP_OUTOFSTOCK_STATUS(SettingSection.LANGUAGE_OPTIONS, "shop-outofstock-status", "&c<Out Of Stock>"),
 
     // Global Options
-    ALLOWED_DIRECTIONS(SettingSection.GLOBAL_OPTIONS, "allowed-directions", new String[]{"DOWN", "WEST", "SOUTH", "EAST", "NORTH", "UP"}),
+    CHEST_DIRECTIONS(SettingSection.GLOBAL_OPTIONS, "chest-directions", new String[]{"BACK", "DOWN", "LEFT", "RIGHT", "UP", "FRONT"}),
```

Comment:
It would probably be good to have a comment explaining what this is

Reply:
I think adding comments into these enums would start getting messy but we can if you think it's necessary. The lang has comments for the config output already. This did remind me to update the allowed-directions in lang to chest-directions though...

Reply:
We can leave it for now and add a comment if people are confused